includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
So what I suggested initialy was to have variable that is the only source of truth. Not merging that with any other list.